CMakeLists.txt

```diff
@@ -1,56 +1,78 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.0.0)
 
-# define the project
-project(nlohmann_json VERSION 2.1.1 LANGUAGES CXX)
+##
+## PROJECT
+## name and version
+##
+project(nlohmann_json VERSION 2.1.1)
```

Comment:
because it is enabled by default, so no need for that. see https://cmake.org/cmake/help/v3.0/command/project.html